server/index.js

```diff
@@ -22,7 +22,7 @@ app.use(jsonMiddleware);
 
 app.use(staticMiddleware);
 
-app.get('/api/search', (req, res, next) => {
+app.get(('/api/search'), (req, res, next) => {
```

Comment:
I've noticed that you put parentheses around your url paths everywhere. it's not that big of a deal, but it just looks funny. nobody really does that